backend/kaos_backend/util/tests/test_validators.py

```diff
@@ -96,6 +96,17 @@ def test_validate_bundle_structure_missing_model_directory():
             BundleValidator.validate_bundle_structure(temp_dir, [])
 
 
+def test_validate_bundle_structure_missing_shebang_line_in_train():
+    with pytest.raises(InvalidBundleError,
+                       match="The train file cannot be executed. Please add the line '#!/usr/bin/xenv python3' "
```

Comment:
why are we restricting ourselves to `python3`?!?

let's keep it more generic - https://en.wikipedia.org/wiki/Shebang_(Unix)

Reply:
Validation now only performed for the existence of shebag